addons/stock/static/tests/tours/stock_report_tests.js

```diff
@@ -96,3 +96,33 @@ registry.category("web_tour.tours").add("test_context_from_warehouse_filter", {
         },
     ],
 });
+
+registry.category("web_tour.tours").add("test_forecast_repenishment", {
+    steps: () => [
+        {
+            trigger: ".o_kanban_record:contains(Lovely product)",
+            run: "click",
+        },
+        {
+            trigger: "button[name=action_product_tmpl_forecast_report]",
+            run: "click",
+        },
+        {
+            trigger: "button.o_forecasted_replenish_btn",
+            run: "click",
+        },
+        {
+            trigger: "button[name=launch_replenishment]",
+            run: "click",
+        },
+        {
+            trigger: ".o_web_client:not(:has(.modal-dialog))",
+            run() {},
+        },
```

Comment:
```suggestion
        {
            trigger: ".o_web_client:not(:has(.modal-dialog))"
        },
```
Pretty sure you don't have to do an empty `run()`.